python/clean_content.py

```diff
@@ -142,9 +142,9 @@ def clean_content():
 
             textdata = TextData()
             metadata = Metadata()
-            progress_bar = jenkins_compatible_progress_bar()
+            # progress_bar = jenkins_compatible_progress_bar()
 
-            for content_item in progress_bar(items_from_content_file()):
+            for content_item in tqdm(items_from_content_file()):
```

Comment:
It would be a little bit clearer to separate the progress bar related changes out from the writerow change?

Reply:
I've taken out the tqdm change now as I don't think it is compatible with Jenkins either.